client/app/bundles/course/survey/constants.js

```diff
@@ -1,9 +1,9 @@
 import mirrorCreator from 'mirror-creator';
 
 export const questionTypes = {
-  TEXT: '0',
-  MULTIPLE_CHOICE: '1',
-  MULTIPLE_RESPONSE: '2',
+  TEXT: 'text',
```

Comment:
If I don't this, it seems I have to manually convert `question_type` to an integer in the controller, otherwise, I'll get 

```
ArgumentError ('0' is not a valid question_type)
```